eng/config/globalconfigs/Common.globalconfig

```diff
@@ -11,7 +11,7 @@ dotnet_diagnostic.CA2009.severity = warning
 dotnet_diagnostic.CA2012.severity = warning
 
 # IDE0055: Fix formatting
-dotnet_diagnostic.IDE0055.severity = warning
+dotnet_diagnostic.IDE0055.severity = none
```

Comment:
Will file issue to reenable this once we can once the formatting analyzer is fixed up to support collection expressions.